CoreScriptsRoot/LoadingScript.lua

```diff
@@ -560,7 +552,9 @@ renderSteppedConnection = game:GetService("RunService").RenderStepped:connect(fu
 	end
 end)
 
-spawn(function()
+coroutine.wrap(function()
+	coroutine.yield()
```

Comment:
The wrap/yield combo is there to keep the yielding behavior of spawn() without actually pausing until the first wait resume.
It can be removed if that block doesn't depend on the yield; that's something I'll test.